proxy/envoy/discovery.go

```diff
@@ -226,6 +226,11 @@ func NewDiscoveryService(ctl model.Controller, configCache model.ConfigStoreCach
 		configCache.RegisterEventHandler(model.IngressRule.Type, configHandler)
 		configCache.RegisterEventHandler(model.EgressRule.Type, configHandler)
 		configCache.RegisterEventHandler(model.DestinationPolicy.Type, configHandler)
+
+		configCache.RegisterEventHandler(model.HTTPAPISpec.Type, configHandler)
+		configCache.RegisterEventHandler(model.HTTPAPISpecBinding.Type, configHandler)
+		configCache.RegisterEventHandler(model.QuotaSpec.Type, configHandler)
+		configCache.RegisterEventHandler(model.QuotaSpecBinding.Type, configHandler)
```

Comment:
can you add a note here saying that this invalidation is too coarse grained? API spec changes result in mixer filter changes. But it does not mean we have to blow up everything in the cache.